src/Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: dia
+.PHONY: math
+
+default:
+	@echo "Type one of the following:"
+	@echo "  make dia                   for the diagrams"
+	@echo "  make math                  for the math formulas"
+	@echo "  make clean                 remove useless files"
+
+dia: dia/*.dia
+	@echo "Producing diagrams..."
+	@make install -C dia
+
+math: math/*.tex
+	@echo "Producing math formulas..."
+	@make install -C math
```

Comment:
I get an error when executing `make math` in `src` folder.
But it works fine if I execute `make` directly in the `math` folder.